omod/src/main/webapp/fragments/scheduleAppointment.gsp

```diff
@@ -93,7 +93,7 @@ ${ ui.includeFragment("appointmentschedulingui", "timeZoneWarning") }
                </daterangepicker>
 
                 <% if (canOverbook) { %>
-                   <div id="selectIncludeSlotsThatRequireOverbook" class="inlineBox">
+                   <div class="inlineBox">
```

Comment:
thanks @NehalJain159  how do things appear on minimizing and maximizing the page

Reply:
@HerbertYiga ,I have added the ss for big and medium screeen respectively as the 2 attachments in the initial pullRequest.Could you please review it and for small screen the ss is atached below
![SmallScreen](https://user-images.githubusercontent.com/55805335/118945007-b17b7300-b972-11eb-9771-5e41a5cc16d7.png)